src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -2543,30 +2543,23 @@ public static void DCJS_VerifyDateTimeForFormatStringDCJsonSerSettings()
         dcjsSettings = new DataContractJsonSerializerSettings() { DateTimeFormat = jsonTypes.DTF_DMMMM };
         var actual3 = SerializeAndDeserialize(graph, "{\"DateTime\":\"1, mayo\",\"OffsetMinutes\":60}", dcjsSettings);
         Assert.NotNull(actual3);
-        Assert.True(actual3 == new DateTimeOffset(2017, 5, 1, 1, 0, 0, new TimeSpan(1, 0, 0)));
+        var expected3 = new DateTimeOffset(DateTime.Now.Year, 5, 1, 0, 0, 0, new TimeSpan(1, 0, 0));
+        Assert.True(actual3 == expected3, 
```

Comment:
.... more useful _how_?  Just because of the variable name?  It otherwise generates a similar message:

```
Expected: 2017-04-26T17:31:30.6734784+00:00
Actual:   2017-04-25T10:31:30.6734784-07:00
```

... plus the line number, so you can find the offending test.